lib/rubocop/cop/rspec/verified_double_reference.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for consistent verified double reference style.
+      #
+      # @see https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles
+      #
+      # This cop can be configured in your configuration using the
+      # `EnforcedStyle` option and supports `--auto-gen-config`.
+      #
+      # @example `EnforcedStyle: const`
+      #   # bad
+      #   let(:foo) do
+      #     instance_double('ClassName', method_name: 'returned_value')
+      #   end
+      #
+      #   # good
+      #   let(:foo) do
+      #     instance_double(ClassName, method_name: 'returned_value')
+      #   end
+      #
+      # @example `EnforcedStyle: string`
+      #   # bad
+      #   let(:foo) do
+      #     instance_double(ClassName, method_name: 'returned_value')
+      #   end
+      #
+      #   # good
+      #   let(:foo) do
+      #     instance_double('ClassName', method_name: 'returned_value')
+      #   end
+      class VerifiedDoubleReference < Base
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Prefer `%<style>s` references for verfied doubles.'
+
+        SUPPORTED_STYLES = %w[const string].freeze
```

Comment:
I don’t think this constant does anything (and we need to remove it from a couple of our existing gems). I think the `ConfigurableEnforcedStyle` module goes looking in the `SupportedStyles` part of the cop’s configuration.

Reply:
Done